### ISSUE TITLE
Will this EFI work with Intel i5-1135G7 processor?

### ISSUE THREAD

**Issue by abhinavbharadwajr** [leon0410898/OC-XPS13-9300#30]
Will this possible fix work on i5-1135G7 Core having the same Intel iris Xe Graphics. As you might be aware both Intel i7-1165G7 and i5-1135G7 are from the same Family of Tiger Lake processor ([check here](https://en.wikipedia.org/wiki/Tiger_Lake#Mobile_processors_(UP3-class))).

**Comment by takeruadelbert** [comment]
hi @abhinavbharadwajr , any update regarding Intel iris Xe?

**Comment by abhinavbharadwajr** [comment]
> hi @abhinavbharadwajr , any update regarding Intel iris Xe?

Nope, things are not Working the Way we would expect with Intel Iris Xe iGPUs. But, many have found ways with works around, yet I couldn't with my machine.

**Comment by alperenfatiih** [comment]
Any progress intel xe graphics

**Comment by fisherlin007** [comment]
it works now?
